libs/pika/async_mpi/tests/unit/mpi_ring_async_sender_receiver.cpp

```diff
@@ -279,7 +276,7 @@ struct message_receiver
                 // the recursive lambda will handle it
                 auto rx_snd2 = ex::just(buf, message_size, MPI_UNSIGNED_CHAR, prev_rank(rank, size),
                                    tag, MPI_COMM_WORLD) |
-                    mpi::transform_mpi(MPI_Irecv, mpi::stream_type::receive_2) |
+                    mpix::transform_mpi(MPI_Irecv /*, mpix::stream_type::receive_2*/) |
```

Comment:
Can the `stream_type`s be uncommented?

Reply:
same as above. stream throttling disabled, so commented out.

Do I remove all throttling bits and put it back in a cleaned up separate PR?